### PR TITLE
Ddcyls 4648 - Validation of non-UK bank account number does not allow spaces

### PR DESCRIPTION
**Pull request by AthishThayalan**
[Your pull request description here]

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [ ] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [ ] Appropriate labels added.
- [ ] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/forms/mappings/AddressMappings.scala [hunk 1]
```diff
@@ -45,6 +45,7 @@ trait AddressMappings extends Mappings {
   protected def postcodeOrCountryMapping(isUKAddress: Boolean): (String, Mapping[String]) = {
     if (isUKAddress) {
       "postCode" -> text("error.required.postcode")
+        .transform[String](_.trim,identity)
```
Comment (jpatton638) [inline app/forms/mappings/AddressMappings.scala:48]: You can do the same thing you have done above here and remove the `.transform` call

File: paces" in {
Comment (jpatton638) [inline paces" in {:1]: This file needs removing

**Review by jpatton638 — CHANGES_REQUESTED** [review]

File: app/forms/mappings/AddressMappings.scala (hunk 1 above)
Comment (AthishThayalan) [reply app/forms/mappings/AddressMappings.scala]: Resolved

File: paces" in {
Comment (AthishThayalan) [reply paces" in {:1]: Resolved

**Review by jpatton638 — APPROVED** [review]

**Review by VJWatson — APPROVED** [review]

**Review by jpatton638 — APPROVED** [review]

**Comment by VJWatson** [comment]
As a Question, i've seen on here we've got a test for max length, is there one needed to prove it would fail going under this length as well?

**Pull request merged**